### ISSUE TITLE
Initial error = 2 = out of memory

### ISSUE THREAD

**Issue by jcldc** [exoclime/HELIOS-K#9]
Hi,
I am not able to complete the test run. I got an "Initial error = 2 = out of memory" error (see below).

My configuration is :  

- Cuda 10.1
- try to compil and run with 
  make SM=30
  and
 make SM=75
- GPU nvidia rtx 2080TI. 
This card has 11GB only, could it be an issue ?

 Thanks in advance for your help.
 
 Jean-Charles

PS:  here is the output

```
[lacuna@node50 HELIOS-K-master]$ ./heliosk
There is 1 CUDA Device
Name:GeForce RTX 2080 Ti, Major:7, Minor:5, Max threads per Block:1024, Max x dim:1024
, #Multiprocessors:68, Clock Rate:1545000, Memory Clock Rate:7000000, Global Memory:11554717696, Shared memory per block: 49152
1000 0 0.001 1e+06

Version: 2.04
Using device 0

Runtime Version 10010
Driver Version 10020
GIT Describe: 
Build Date: Mon Sep 14 16:49:01 CEST 2020
Build Path: /net/GSP/users/lacuna/HELIOS-K-master
Build System: Linux node50.lam.fr 3.10.0-1062.9.1.el7.x86_64 #1 SMP Fri Dec 6 15:49:49 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
Build Compute Capability: SM=30

name = i
T = 100
P = 1
cia System = -
pathToData = data/
numin = 0
numax = 1000
dnu = 0.001
Nnu per bin = 200000
Number of points: 1000000
cutMode = 0
cut = 25
doResampling = 0
nC = 20
doTransmission = 0
nTr = 1000
dTr =  0.05
doStoreFullK = 1
pathToK = 
dostoreK = 0
nbins = 5
kmin = 0
qalphaL = 1
gammaF = 1
doMean = 0
Units = 0
Replace files = 1
profile = 1
doTuning = 1
def_TOL = 1.43e-17
def_TOLf = 2.48e-12
def_nthmax = 1048576
def_nlmax = 32768
def_maxlines = 1048576
def_maxfiles = 500
def_NmaxSample = 100

Read Molecule param file: |01_hit16| 0 1
mean mass 18.0153
Read Molecule param file: |01_hit16| 0 1
Species Name = 01_hit16
dataBase = 0
Molecule Number = 1
default L = 0
default n = 0

Read partition function: data/q1.txt
Q 0 T0: 99, T1: 100, q0: 34.6424, q1: 35.1531, q 35.1531
Read partition function: data/q2.txt
Q 1 T0: 99, T1: 100, q0: 34.9291, q1: 35.4441, q 35.4441
Read partition function: data/q3.txt
Q 2 T0: 99, T1: 100, q0: 208.762, q1: 211.84, q 211.84
Read partition function: data/q4.txt
Q 3 T0: 99, T1: 100, q0: 169.885, q1: 172.414, q 172.414
Read partition function: data/q5.txt
Q 4 T0: 99, T1: 100, q0: 171.985, q1: 174.547, q 174.547
Read partition function: data/q6.txt
Q 5 T0: 99, T1: 100, q0: 1025.95, q1: 1041.22, q 1041.22
Read partition function: data/q129.txt
Q 6 T0: 99, T1: 100, q0: 200.436, q1: 203.437, q 203.437
mean mass 18.0153, Sscale 1
Initial error = 2 = out of memory

```

**Comment by sigrimm** [comment]
Hi, 
The issue should be fixed in version 2.05. It occurred because of a missing initialization of HITRAN type buffer sizes.

Could you retry to run the code? 

Best regards,
Simon

**Comment by jcldc** [comment]
Hi,
thank you it works now !
Jean-Charles
